lightkurve/correctors/pldcorrector.py

```diff
@@ -82,199 +116,509 @@ class PLDCorrector(object):
     .. [3] Luger et al. (2018), ads:2018AJ....156...99L
         (arXiv:1702.05488)
     .. [4] EVEREST pipeline webpage, https://rodluger.github.io/everest
+    .. [5] Celerite documentation, https://celerite.readthedocs.io/en/stable/
+    .. [6] Exoplanet documentation, https://exoplanet.readthedocs.io/en/stable/
+    .. [7] PyMC3 documentation, https://docs.pymc.io
+
+    Parameters
+    ----------
+    tpf : `TargetPixelFile` object
+        The pixel data from which a light curve will be extracted.
+    aperture_mask : 2D boolean array or str
+        The pixel aperture mask that will be used to extract the raw light curve.
+    pld_aperture_mask : 2D boolean array or str
```

Comment:
This keyword is not very descriptive. How about `design_matrix_mask`.

Reply:
That makes it sound like we're masking the design matrix itself instead of using an aperture mask on pixels. Maybe `design_matrix_aperture_mask`? It's longer but more descriptive. I'll change it to that for now, let me know if you have other suggestions.